tensorboard/plugins/profile/tf_op_profile/utils.ts

```diff
@@ -80,4 +88,11 @@ namespace tf_op_profile {
       ? '0.00%'
       : (fraction * 100).toPrecision(2) + '%';
   }
+  export function timeWasted(node: any) {
+    if (!node || !node.metrics) return 0 / 0;
```

Comment:
what's the purpose of 0/0? Did you mean to write `NaN`?

Reply:
Done.